### ISSUE TITLE
Are specific sensory experiences subject to WCAG 1.2.1?

### ISSUE THREAD

**Issue by gnpivo** [w3c/wcag#3118]
Hello! I have been trying to understand the accessibility requirements for "specific sensory experiences" as defined by the WCAG specification: 
     
> ***specific sensory experience***
>
> a sensory experience that is not purely decorative and does not primarily convey important information or perform a function
>
> > Examples include a performance of a flute solo, works of visual art etc.

If a specific sensory experience existed in a form that was audio-only or video-only (like, say, a performance of a flute solo), would it be necessary to provide an [alternative for time-based media](https://www.w3.org/TR/WCAG/#dfn-alternative-for-time-based-media) to fulfill [Success Criterion 1.2.1: Audio-only and Video-only (Prerecorded)](https://www.w3.org/TR/WCAG/#audio-only-and-video-only-prerecorded)? Or would it be enough to just provide a [text alternative](https://www.w3.org/TR/WCAG/#dfn-text-alternative) as described by [Success Criterion 1.1.1: Non-text Content](https://www.w3.org/TR/WCAG/#text-alternatives)?

**Comment by gnpivo** [comment]
Thanks for your reply. I see what you mean about "specific sensory experiences" being used as an exception to SC 1.1.1, but I still think there may be an issue here. When you say:

> It is just not a normative requirement to provide an alternative for stuff that cannot be rendered in words in an equivalent way.

Does that mean that "specific sensory experiences" are not required to have a text alternative at all? I interpret the spec as requiring there to be at least *some* text alternative, even if the content cannot be rendered in words in an equivalent way:

> All [non-text content](https://www.w3.org/TR/WCAG/#dfn-non-text-content) that is presented to the user has a [text alternative](https://www.w3.org/TR/WCAG/#dfn-text-alternative) that serves the equivalent purpose, except for the situations listed below.
>
> ...
>
> - **Sensory:** If non-text content is primarily intended to create a [specific sensory experience](https://www.w3.org/TR/WCAG/#dfn-specific-sensory-experience), then *text alternatives at least provide descriptive identification of the non-text content*.

(*Emphasis* added by me.)

---

My original issue was whether or not it's possible for this sort of content to be subject to other criteria at the same time (specifically 1.2.1). In particular, 1.2.1 puts specific requirements on all prerecorded audio-only and video-only media:


> For [prerecorded](https://w3c.github.io/wcag/21/guidelines/#dfn-prerecorded) [audio-only](https://w3c.github.io/wcag/21/guidelines/#dfn-audio-only) and prerecorded [video-only](https://w3c.github.io/wcag/21/guidelines/#dfn-video-only) media, the following are true, except when the audio or video is a [media alternative for text](https://w3c.github.io/wcag/21/guidelines/#dfn-media-alternative-for-text) and is clearly labeled as such:
>
> **Prerecorded Audio-only**
>
> An [alternative for time-based media](https://w3c.github.io/wcag/21/guidelines/#dfn-alternative-for-time-based-media) is provided that presents equivalent information for prerecorded audio-only content.
> **Prerecorded Video-only**
>
> Either an alternative for time-based media or an audio track is provided that presents equivalent information for prerecorded video-only content.

The example of a flute solo is distinct from an oil painting here because the flute solo would generally take the form of prerecorded audio-only media (although I suppose it could be a combination of video and audio and thereby potentially be subject to other criteria instead???).
